### PR TITLE
ENDOC-311 update individual CRD descriptions

### PR DESCRIPTION
**Pull request by joewhite101**
The goal of this PR is to simplify the descriptions provided in the CRD field descriptions and to link to docs for assumptions and detailed discussions. this should be viewed in concert with this docs PR where the deeper information has been migrated to docs.

https://github.com/entando/entando-docs/pull/288

This is also pending confirmation/testing that the field level descriptions support markdown for the links.

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-operator-bundle).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-operator-bundle/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by entando-jx — APPROVED** [review]

File: manifests/entandoapps.entando.org.crd.yaml
Comment (nshaw) [inline manifests/entandoapps.entando.org.crd.yaml:50]: Might be good to use preferred capitalization for MySQL and PostgreSQL

File: manifests/entandoapps.entando.org.crd.yaml
Comment (nshaw) [inline manifests/entandoapps.entando.org.crd.yaml:66]: We're going to have to keep in mind we have version-specific links in the CRDs. Do we have a README or doc somewhere that documents our release instructions for the operator? We might need a reminder there to review these kinds of links as we're releasing new versions of the operator. I have something like that in the entando-docs README which I review when we cut a new version.

File: manifests/entandoapps.entando.org.crd.yaml
Comment (nshaw) [inline manifests/entandoapps.entando.org.crd.yaml:73]: Do we document the WAR deployment setup somewhere? Not finding it in docs and  I wonder if we need something around controlling/changing the web context path itself. I can't recall if we had a ticket for that before we lost Jira but I know I've heard mention of needing to do that, switching out from entando-de-app and/or removing it completely. Just asking here if we need to get that in queue.

File: manifests/entandoapps.entando.org.crd.yaml
Comment (nshaw) [inline manifests/entandoapps.entando.org.crd.yaml:79]: Is it in the same namespace in the case of  the cluster-scoped operator?  I think the tutorial had the secrets in the openshift-operator namespace but I don't know if the secrets get copied over as well?

File: manifests/entandocompositeapps.entando.org.crd.yaml
Comment (nshaw) [inline manifests/entandocompositeapps.entando.org.crd.yaml:40]: "If Oracle is selected, the database instance must be created manually since the operator does not deploy Oracle instances."

File: manifests/entandodatabaseservices.entando.org.crd.yaml
Comment (nshaw) [inline manifests/entandodatabaseservices.entando.org.crd.yaml:43]: Add embedded to the list?

File: manifests/entandodatabaseservices.entando.org.crd.yaml
Comment (nshaw) [inline manifests/entandodatabaseservices.entando.org.crd.yaml:44]: Same comment re this Oracle sentence as above.

File: manifests/entandokeycloakservers.entando.org.crd.yaml
Comment (nshaw) [inline manifests/entandokeycloakservers.entando.org.crd.yaml:56]: Ditto on language for Oracle

File: manifests/entandoplugins.entando.org.crd.yaml
Comment (nshaw) [inline manifests/entandoplugins.entando.org.crd.yaml:83]: TODO?  You could point to the tutorial on the operator since I have the details on creating secrets there.

Comment (nshaw) [reply manifests/entandoplugins.entando.org.crd.yaml:83]: This should be the URL to the tutorial page - https://dev.entando.org/v6.3/tutorials/devops/installation/open-shift/openshift-install-by-operator-hub.html

File: manifests/entandoplugins.entando.org.crd.yaml
Comment (nshaw) [inline manifests/entandoplugins.entando.org.crd.yaml:184]: Missing space after container.

**Review by nshaw — COMMENTED** [review]
Finished my review, would approve if needed since mostly just suggestions/wordsmithing. The only major is the TODO in there.

**Pull request merged**
